### PR TITLE
added information when comment was edited

### PR DESCRIPTION
**Pull request by zilvinasjuc**
#57

### CODE REVIEW

**Comment by zilvinasjuc** [comment]
#57

File: src/lang/en.json [hunk 1]
```diff
@@ -351,17 +354,16 @@
       "Forbidden": "You are trying to access forbidden resources",
       "NotFound": "Not found",
       "CommentTooLong": "Comment lenght limit was reached",
-      "RemoteAccessPointNotSet": "Remote access point is not set.",
-      "InvalidTags": "Invalid tags",
```
Comment (dariusdev) [inline src/lang/en.json]: "RemoteAccessPointNotSet","InvalidTags" why deleted?

File: src/lang/en.json [hunk 2]
```diff
@@ -132,10 +133,7 @@
         "tableTooSmall": "Minimum size of the table is 1x1",
         "emptyModel": "Model table is required",
         "emptyModelInput": "Make sure you have filled all table fields",
-        "titleTooLong": "Maximum 64 characters",
-        "invalidTag": "Invalid tag: only letters, numbers and \"-\" symbols are allowed",
```
Comment (dariusdev) [inline src/lang/en.json]: "invalidTag": "Invalid tag: only letters, numbers and \"-\" symbols are allowed",	
"invalidTagLength": "Invalid tag: tag is too short",	
"invalidTagCount ": "Maximum tag limit is 5" deleted. Why?

File: src/lang/en.json [hunk 3]
```diff
@@ -14,7 +14,6 @@
         "upload": "Upload dataset",
         "load": "Continue",
         "title": "Title",
-        "tags": "Tags",
```
Comment (dariusdev) [inline src/lang/en.json]: deleted?

**Pull request closed without merging**